cmd/celestia-appd/cmd/init.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"github.com/cosmos/cosmos-sdk/x/genutil/client/cli"
+	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-app/v4/app"
+)
+
+// InitCmd returns a custom version of the default init command with improved help docs.
+func InitCmd(capp *app.App) *cobra.Command {
+	cmd := cli.InitCmd(capp.BasicManager, app.DefaultNodeHome)
+
+	cmd.Short = "Initialize configuration files for a Celestia consensus node"
+	cmd.Long = "This command sets up a genesis file and default configuration for either a validator or a full node."
```

Comment:
Again I don't think we need to differentiate this part

```suggestion
	cmd.Long = "This command creates a genesis file and the default configuration files for a consensus node."
```